packages/action-bar/src/ActionBar.ts

```diff
@@ -71,6 +75,7 @@ export class ActionBar extends SpectrumElement {
     public override render(): TemplateResult {
         return html`
             <sp-popover ?open=${this.open} id="popover">
+                <sp-close-button class="close-button"></sp-close-button>
```

Comment:
@pfulton it appears that the CSS implementation for this does not have an accessible label. Can we get clarify on what that should be "Close", "Deselect", etc.?

Reply:
Did we ever get content for this button, @pfulton?